systemtests/scripts/test_func.sh

```diff
@@ -270,3 +271,29 @@ function check_if_ansible_ready() {
         sudo yum -y install --enablerepo=epel ansible
     fi
 }
+
+function get_openshift_version() {
+    echo $(oc version | sed -nre '/^oc/s/^.*v(([0-9]+\.)*[0-9]+).*$/\1/p')
+}
+
+function get_kubeconfig_path() {
+    OC_39='/var/lib/origin/openshift.local.config/master/admin.kubeconfig'
+    OC_310='./openshift.local.clusterup/kube-apiserver/admin.kubeconfig'
+
+    if [[ $(get_openshift_version) == '3.9.0' ]]; then
```

Comment:
Maybe match on major.minor, not major.minor.micro. Ideally we should track the latest micro version anyway, wdyt?

Reply:
Makes sense, but do you think that openshift team will release some new micro version od 3.9? Anyway I will update it to match only major.minor, thanks for suggestion 👍